platform/android/java/app/build.gradle

```diff
@@ -1,6 +1,4 @@
 // Gradle build config for Godot Engine's Android port.
-apply from: 'config.gradle'
-
```

Comment:
This is needed when doing gradle builds in the Godot Editor since in that environment, the `app/build.gradle` is the root gradle build file.